tools/js-tools/eslintrc/typescript.js

```diff
@@ -12,5 +12,19 @@ module.exports = {
 		'no-undef': 'off',
 
 		'@typescript-eslint/no-unused-vars': [ 'warn', { argsIgnorePattern: '^_' } ],
+
+		'@typescript-eslint/no-empty-object-type': [
+			'error',
+			{ allowInterfaces: 'with-single-extends' },
+		],
```

Comment:
@anomiex,
This change appears to be causing an ESLint error at the first character of **_Every_** typescript file within the monorepo. 🤔 (In my VSCode editor)
Or is it only me?
See Screenshot:

![Screen Shot 2024-12-07 at 23 19 07](https://github.com/user-attachments/assets/a642d5a8-d8ce-4fa5-b533-b3b90254fb9b)

So then as a result of this ESLint error on every file, any commits I attempt are aborted, unless I override using the `--no-verify` flag.
Prior to this change I was not experiencing this issue, and when I remove this rule entry (lines 16 - 19), the ESLint error on every file goes away.

Can you take a look and let me know if you are experiencing this also, or can reproduce the issue I am describing here?  Thanks! 🙂

Reply:
Oh wait, my apologies, after reinstalling pnpm dependencies, the error seems to have gone away and everything seems to be working as expected.
Disregard, sorry about that! 😊

Reply:
Glad you figured it out! 🙂

For anyone else looking on: The `Definition for rule '@typescript-eslint/no-empty-object-type' was not found` error was because that rule was added in the updated version of the '@typescript-eslint/eslint-plugin` package, so you have to have done a `pnpm install` to get the new package version with the new rule. 🙂